packages/dev/loaders/src/glTF/glTFFileLoader.ts

```diff
@@ -434,7 +451,7 @@ export class GLTFFileLoader extends GLTFLoaderOptions implements IDisposable, IS
      */
     public constructor(options?: Partial<Readonly<GLTFLoaderOptions>>) {
         super();
-        this.copyFrom(options);
+        this.copyFrom(Object.assign({ ...GLTFLoaderDefaultOptions }, options));

```

Comment:
does this work?

```suggestion
        this.copyFrom({ ...GLTFLoaderDefaultOptions, options }));
```

Reply:
Well, I would have to check if options is undefined, so maybe something like:
```ts
this.copyFrom(options ? {...GLTFLoaderDefaultOptions, ...options} : GLTFLoaderDefaultOptions);
```
Not sure which option is better.